docs/source/glossary.md

```diff
@@ -17,4 +17,20 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
-:::::
+
+[Equidispersion](http://www.ce.memphis.edu/7012/L20_CountDataModels_v2.pdf)
+  If in a Poisson distribution if the variance equals the mean of the distribution, it is reffered to as equidispersion.
+
+[Generalized Poisson PMF](https://www.sciencedirect.com/science/article/pii/S0047259X14000256)
+  A generalization of the {term}`Poisson distribution`, with two parameters X1, and X2, is obtained as a limiting form of the {term}`generalized negative binomial distribution`. The variance of the distribution is greater than, equal to or smaller than the mean according. as X2 is positive, zero or negative. For formula and more detail visit the link 
+
+[Bayes' theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem)
+  Describes the probability of an event, based on prior knowledge of conditions that might be related to the event. For example, if the risk of developing health problems is known to increase with age, Bayes' theorem allows the risk to an individual of a known age to be assessed more accurately (by conditioning it on their age) than simply assuming that the individual is typical of the population as a whole.
+
+[Markov Chain](https://setosa.io/ev/markov-chains/)
+  A Markov chain or Markov process is a stochastic model describing a sequence of possible events in which the probability of each event depends only on the state attained in the previous event. For a visual explantation, visit the link.
+
+[Markov Chain Monte Carlo](https://en.wikipedia.org/wiki/Markov_chain_Monte_Carlo)
+[MCMC](https://machinelearningmastery.com/markov-chain-monte-carlo-for-probability/)
```

Comment:
Let's leave only one title (and one title link) for Markov Chain Monte Carlo, which can look like this:
[Markov Chain Monte Carlo (MCMC)]

Reply:
I prefer very much to have the two titles, especially thinking about linking to that term, but I agree both should have the same link.

If we use `Markov Chain Monte Carlo (MCMC)` we have to use

```
{term}`Markov Chain Monte Carlo (MCMC)` 
```

to link to it. Otherwise we should be able to use both

```
{term}`MCMC`
{term}`Markov Chain Monte Carlo`
```

or maybe only one of them, not completely sure how it works, if both are usable, only the first, only the last... but I do know the glossary allows multiple names per definition.